public/static/plugins/perf/js/trend_chart.js

```diff
@@ -544,7 +590,23 @@ var drawSingleTrendChart = function(params) {
 
     focusedText
       .attr('y', function(d, i) { return opsLabelsY[i] })
-      .text(function (d, i) { return d3.format(',.0f')(values[i])})
+      .text(function (d, i) {
+        var value = values[i];
+        var absolute = Math.abs(value);
+        if ( absolute == 0) {
+          return "0";
+        } else if ( absolute < 1) {
+          if ( absolute >= .1) {
+            return cfg.formatters.digits_1(value);
+          }  else if ( val >= .01) {
+            return cfg.formatters.digits_2(value);
```

Comment:
This looks like code duplication. I'd rather created a separate component for it, Angularjs filter, probably.